APP-MANAGER

```diff
@@ -213,6 +222,11 @@ AMCLIUPPER=$(echo "$AMCLI" | tr '[:lower:]' '[:upper:]')
 # Create new data directory and move important files there
 AMDATADIR="$DATADIR/AM"
 mkdir -p "$AMDATADIR"
+if [ "$sed_cmd" = gsed ]; then
+	mkdir -p "$AMDATADIR/bin"
+	export PATH="${AMDATADIR}"/bin/:"${PATH}"
+	[ ! -f "${AMDATADIR}"/bin/sed ] && ln -s /usr/local/bin/gsed  "${AMDATADIR}"/bin/sed
```

Comment:
`ln -s "$(command -v gsed)" "${AMDATADIR}"/bin/sed`

And likely with this you don't need to patch `gsed` into the install script. 

Also some check to make sure that gsed exists on the system.

Reply:
I need it, since apps are installed with root privileges and scripts runs out of AM itself. AM only executes them, and they rely on variables and commands available system wide.

I have shown a lot of screenshots and videos where installing an app returns a `sed` error after the download. This patch solves this.

I use AM as root, so commands available locally are only for local apps, not for system apps.

Reply:
> I need it, since apps are installed with root privileges and scripts runs out of AM itself. AM only executes them, and they rely on variables and commands available system wide.
> 
> I have shown a lot of screenshots and videos where installing an app returns a `sed` error after the download. This patch solves this.
> 
> I use AM as root, so commands available locally are only for local apps, not for system apps.

That doesn't mean that you can't `ln -s "$(command -v gsed)" "${AMDATADIR}"/bin/sed` instead of the hardcoded `ln -s /usr/local/bin/gsed  "${AMDATADIR}"/bin/sed`

Reply:
In other words `ln -s /usr/local/bin/gsed  "${AMDATADIR}"/bin/sed` will only work if you have gsed in `/usr/local/bin`, which I don't know how freebsd works but if you install the coreutils package `gsed` should be in `/bin` or `/usr/bin`

Reply:
sorry, I misunderstood. It's a language barrier issue, as you pointed out on Discord.

fixed in https://github.com/ivan-hc/AM/pull/1209/commits/67037cd0716d1c512521f2376cce6bf30ec2b98f